cmd/ebrelayer/relayer/relayer_test.go

```diff
@@ -5,23 +5,41 @@ package relayer
 //
 //    Tests Relayer functionality.
 //
+//		`go test network.go relayer.go relayer_test.go`
 // ------------------------------------------------------------
 
 import (
 	"testing"
+	"fmt"
+	"strings"
+	"encoding/hex"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+	app "github.com/swishlabsco/cosmos-ethereum-bridge"
 )
 
 const (
 	ChainID          = "testing"
-	Client           = "wss://ropsten.infura.io/ws"
+	Socket           = "wss://ropsten.infura.io/ws"
 	ContractAddress  =  "3de4ef81Ba6243A60B0a32d3BCeD4173b6EA02bb"
-	// EventSig is hash of "LogLock(bytes32,address,bytes,address,uint256,uint256)"
 	EventSig         = "0xe154a56f2d306d5bbe4ac2379cb0cfc906b23685047a2bd2f5f0a0e810888f72"
-	Validator        = "cosmos1xdp5tvt7lxh8rf9xx07wy2xlagzhq24ha48xtq"
-
+	Validator        = "validator"
 )
 
-func StartRelayer(t *testing.T) err {
-	err = go(InitRelayer(ChainID, Client, ContractAddress, EventSig, Validator))
-}
+func TestInitRelayer(t *testing.T) {
+	cdc := app.MakeCodec()
 
+	// Parse the address of the deployed contract
+	bytesContractAddress, err := hex.DecodeString(ContractAddress)
+	if err != nil {
+		fmt.Printf("Invalid contract-address: %v", bytesContractAddress)
+	}
+	contractAddress := common.BytesToAddress(bytesContractAddress)
+
+	err = InitRelayer(cdc, ChainID, Socket, contractAddress, EventSig, Validator)
+
+	//TODO: add validator key processing for relayer init
```

Comment:
yeh, the full test should basically create a new validator and then use it, so that it has the key setup

Reply:
Yeah, was referencing some of the tests from the ethbridge for best practices on the key generation process. Main goal of the PR is fixing CircleCi - then we can upgrade the test suite to be more robust.